packages/neuron-wallet/src/controllers/wallets.ts

```diff
@@ -28,7 +29,7 @@ class WalletsController {
     this.channel = channel
   }
 
-  public static index = (): ChannelResponse<Wallet[]> => {
+  public static index = (): ChannelResponse<WalletData[]> => {
```

Comment:
As #160 in internal discussion, we are going to use method names like `getAll` instead of `index`, `get` instead of `show`

Reply:
ok

Reply:
Which pr?

Reply:
You can find it [here](https://github.com/nervosnetwork/neuron/blob/52d578f64d60895ae97d2b1ff710e2525b4e2d89/packages/neuron-wallet/src/services/networks.ts)